hibike/devices/ExampleDevice/example_device.cpp

```diff
@@ -25,11 +36,23 @@ void loop() {
   // do whatever you want here
   // note that hibike will only process one packet per call to hibike_loop()
   // so exessive delays here will affect hibike.
-  value1 += 1;
-  value2 += 2;
-  value3 += 3;
-  value4 += 4;
-  value5 += 5;
+  //don't change Kumiko.
```

Comment:
As its currently implimented, Kumiko will be changing back and forth every loop cycle.  That's kinda fast.

It's also useful for testing to have at least one variable that only changes if it's been written to.